src/phoenix/config.py

```diff
@@ -331,5 +353,85 @@ def get_web_base_url() -> str:
     return get_base_url()
 
 
+class LoggingMode(Enum):
+    AS_LIBRARY = "default"
+    AS_APPLICATION = "structured"
```

Comment:
```suggestion
    DEFAULT = "default"
    STRUCTURED = "structured"
```

Not sure I see the need for more additional terminology.

Reply:
Done